File: neko-example/src/clojure/com/example/hello_clojure/HelloActivity.clj
(ns com.example.hello_clojure.HelloActivity
  (:gen-class :main false
              :extends android.app.Activity
              :exposes-methods {onCreate superOnCreate})
  (:import [com.example.cool_clojure_app R$layout]))

(defn -onCreate
  [this bundle]
  (doto this
    (.superOnCreate bundle)
    (.setContentView R$layout/main)))

File: neko-example/src/java/com/example/cooljavaapp/HelloActivity.java
package com.example.cooljavaapp;

import android.app.Activity;
import android.os.Bundle;

/**
 * A minimal "Hello, World!" application demonstrating use of string resources.
 */
public class HelloActivity extends Activity {
    /** Called when the activity is first created. */
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // set the layout.
        setContentView(R.layout.main);
    }
}